packages/liveblocks-core/package.json

```diff
@@ -46,6 +46,9 @@
     "showdeps:high-level": "depcruise src --include-only '^src' --exclude='(^src/index.ts|shallow.ts|__tests__)' --collapse='^src/(refs|lib|compat|types|crdts|protocol)' --output-type dot | dot -T svg > /tmp/dependency-graph.svg && open /tmp/dependency-graph.svg"
   },
   "license": "Apache-2.0",
+  "dependencies": {
+    "nanoid": "^3"
+  },
```

Comment:
As I mentioned to you already, I'd prefer to vendor in `nanoid` at this point if the license allows it, or implement a similar function (it's really small) ourselves, rather than to introduce this as a dependency. There is lots of pain with the dependencies that it's worth it, like the ESM/CJS stance/issues that v4 and v5 are causing which is causing us to keep it pinned at v3, the workarounds for the jest config etc etc. It's just a function that shouldn't cause all these issues.

Reply:
I inlined nanoid v3.3.7 and also removed the dependency from our other packages.